clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/BaseIntegrationSpec.scala

```diff
@@ -101,15 +185,17 @@ abstract class BaseIntegrationSpec(clioDescription: String)
 
   /**
     * Run a command with arbitrary args through the main clio-client.
-    *
-    * Fails the test immediately if the client exits early.
+    * Returns a failed future if the command exits early.
     */
   def runClient(command: String, args: String*): Future[HttpResponse] = {
     clioClient
       .instanceMain(
         (Seq("--bearer-token", bearerToken.token, command) ++ args).toArray
       )
-      .fold(err => fail(s"Command exited early with $err"), identity)
+      .fold(
+        err => Future.failed(new Exception(s"Command exited early with $err")),
```

Comment:
If `err` is an exception it should be passed to the `Exception()` constructor so you don't lose the original stack crawl.

Reply:
`err` is an [`EarlyReturn`](https://github.com/broadinstitute/clio/blob/develop/clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala#L42) here, which will only happen if we screw up the args. I'll rename it to be more clear that it's not an exception.